django/template/context.py

```diff
@@ -245,12 +245,12 @@ def new(self, values=None):
         return new_context
 
 
-def make_context(context, request=None):
+def make_context(context, request=None, **kwargs):
     """
     Create a suitable Context from a plain dict and optionally an HttpRequest.
     """
     if request is None:
-        context = Context(context)
+        context = Context(context, **kwargs)
```

Comment:
>
> In django/template/context.py
> <https://github.com/django/django/pull/5610#discussion_r44223132>:
>
> >      """
> >      Create a suitable Context from a plain dict and optionally an HttpRequest.
> >      """
> >      if request is None:
> > -        context = Context(context)
> > +        context = Context(context, **kwargs)
>
> It seems this only handles the request=None case; what about when a
> request is passed in?
>
Ah, I thought maybe RequestContext intentionally didn't have an autoescape
attribute, but it's just a subclass, so obviously that's not right.

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/django/django/pull/5610/files#r44223132>.
>